hyprctl/main.cpp

```diff
@@ -423,6 +424,8 @@ int main(int argc, char** argv) {
         request(fullRequest, 1);
     else if (fullRequest.contains("/keyword"))
         request(fullRequest, 2);
+    else if (fullRequest.contains("/decoration"))
```

Comment:
mismatch: help says decorations, here it's decoration? Later in hyprland it's decorations again?

Reply:
done